lib/compute/region.js

```diff
@@ -215,6 +221,198 @@ Region.prototype.createAddress = function(name, options, callback) {
 };
 
 /**
+ * Get a list of subnetworks in this region.
+ *
+ * @resource [Subnetworks Overview]{@link https://cloud.google.com/compute/docs/subnetworks}
+ * @resource [Subnetworks: list API Documentation]{@link https://cloud.google.com/compute/docs/reference/latest/subnetworks}
+ *
+ * @param {object=} options - Subnetwork search options.
+ * @param {boolean} options.autoPaginate - Have pagination handled
+ *     automatically. Default: true.
+ * @param {string} options.filter - Search filter in the format of
+ *     `{name} {comparison} {filterString}`.
+ *     - **`name`**: the name of the field to compare
+ *     - **`comparison`**: the comparison operator, `eq` (equal) or `ne`
+ *       (not equal)
+ *     - **`filterString`**: the string to filter to. For string fields, this
+ *       can be a regular expression.
+ * @param {number} options.maxApiCalls - Maximum number of API calls to make.
+ * @param {number} options.maxResults - Maximum number of firewalls to return.
+ * @param {string} options.pageToken - A previously-returned page token
+ *     representing part of the larger set of results to view.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {module:compute/subnetworks} callback.subnetworks - Subnetwork objects
+ *     from your project.
+ * @param {?object} callback.nextQuery - If present, query with this object to
+ *     check for more results.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * gce.getSubnetworks(function(err, subnetworks) {
```

Comment:
These examples should should `region.getSubnetworks` instead of `gce.getSubnetworks`.